### DISCUSSION TITLE
[Share]: pkg-size-1s

### DISCUSSION THREAD

**Discussion by mefengl** [privatenumber/pkg-size.dev#13] [Show and tell]
To share an idea, it is usually difficult for people to consider opening this website to check the size. However, people often visit their npm.js page, so it would be cool to add a button there.

Actually, I quickly wrote one. It looks like this:

![demo](https://github.com/privatenumber/pkg-size.dev/assets/71683364/78f3af7a-d991-4b28-92ea-889c78ba0127)

You can find the source code here: https://github.com/mefengl/userscript-pkg-size-1s

**Comment by privatenumber** [comment]
Thanks for sharing, looks very cool!

Do you have any ideas on how pkg-size.dev can be improved in this aspect?

(I moved this from an Issue to a Discussion since there didn't seem to be anything actionable for me.)

**Reply by mefengl** [reply]
Yes, it would definitely be better to have this in discussions.

For this script pkg-size-1s, currently it works on the npm.js package page. However, later I will make it work in a tricky way on the GitHub repo page so that people can easily visit this website.

Regarding pkg-size.dev, I think having a README card that can be used in people's README would be cool. When others click on that card, they can jump to pkg-size.dev.

I saw this:

![image](https://github.com/privatenumber/pkg-size.dev/assets/71683364/c260df49-ac44-4c17-a448-772a63b8fa4a)

But what I mean is something not like a badge (badge is also great); more like a card often seen in GitHub profiles or something similar to star history to show the pkg-size.

And these cards (and badges) can be updated by GitHub action.
